contrib/features/TEMPLATE.md

```diff
@@ -0,0 +1,90 @@
+# Change Request
+
+<!--
+This template is used to propose and discuss major new features to be added to Podman, Buildah, Skopeo, Netavark, and associated libraries.
+Before major features are implemented, a pull request should be opened against the Podman repository with a completed version of this template.
+Discussion on the feature will occur in the pull request.
+Merging the pull request will constitute approval by project maintainers to proceed with implementation work.
+-->
+
```

Comment:
Do we need an assignee field to make sure who works on this implementation if it is accepted? Doesn't mean anyone must work on it ASAP but I think we need to have a OWNER who is reasonable to get it completed.

And then I guess we need a completed field? Or we git rm the doc instead? I think removing the docs out of the repo makes sense to ensure only active stuff is visible in the repo.

Reply:
thinking about this more ... suppose we have a design doc as a PR, and the person starts working on it while its in review (common practice). And it takes some time to get the PR reviewed and merged, and then immediately the PR for the actual implementation comes in.  As soon as that is merged, we are saying `git rm`.  Are we OK with such a short lifespan?

Reply:
I would delete on feature merge yes. I mean it is in the git history forever for anyone to look it up so I don't think that is a problem.

Reply:
If we're really concerned, maybe provide a list of removed docs and the commit that removed them in the README?